assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -47,16 +47,53 @@ function FontControls({ selectedElements, pushUpdate }) {
   const fontFamily = getCommonValue(selectedElements, 'fontFamily');
   const fontSize = getCommonValue(selectedElements, 'fontSize');
   const fontWeight = getCommonValue(selectedElements, 'fontWeight');
+  const timeout = useRef(null);
 
   const {
     state: { fonts },
-    actions: { getFontWeight, getFontFallback },
+    actions: { maybeEnqueueFontStyle, getFontWeight, getFontFallback },
   } = useFont();
   const fontWeights = useMemo(() => getFontWeight(fontFamily), [
     getFontWeight,
     fontFamily,
   ]);
 
+  const loadFont = useCallback(
+    (family, callback) => {
+      // 0. Check browser support to load fonts using JavaScript
+      if (document?.fonts) {
+        const font = `0 ${family}`;
```

Comment:
The `document.fonts.check/load` should include more things:
 * Font weight. E.g. 700 (for "B") or 900 for "Back" in the drop down. And that should look inside the content.
 * If italic is used, this should also include italic.
 * The text content itself (for unicode characters).

Please see the `fonts.check/load` APIs. But this is very important. If the text has italic, it will still jump unless we include it here.